db/migrate/20180507115647_create_iterations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateIterations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :iterations do |t|
+      t.datetime :iteration_time
+      t.integer :iteration_cost, limit: 8
```

Comment:
Not so good names for the fields
`time` and `cost` already belong to an iteration, I think it's better to remove the `iteration_` prefix

Reply:
I agree, please rename these fields to `time` and `cost` instead of `iteration_time` and `iteration_cost`.